src/main/java/seedu/pill/util/Ui.java

```diff
@@ -18,4 +18,8 @@ public String getInput() {
         Printer.printSpace();
         return this.sc.nextLine();
     }
+
+    public String getRawInput() {
+        return this.sc.nextLine();
+    }
```

Comment:
Thank you for adding the new getRawInput() method. Great improvement for maintaining a cleaner output.